### ISSUE TITLE
IHME Projections

### ISSUE THREAD

**Issue by JacinthLaval** [starschema/COVID-19-data#211]
IHME's COVID-19 projections were developed in response to requests from the University of Washington School of Medicine and other US hospital systems and state governments working to determine when COVID-19 would overwhelm their ability to care for patients. The forecasts show demand for hospital services, including the availability of ventilators, general hospital beds, and ICU beds, as well as daily and cumulative deaths due to COVID-19.

http://www.healthdata.org/covid/data-downloads

**Data format**
csv

**Comment by JacinthLaval** [comment]
This is Todd Crosslin from Snowflake.  JacinthLaval is my personal GitHub account

**Comment by chrisvoncsefalvay** [comment]
Table `IHME_COVID_19` has been set up within `COVID19_DEV.PUBLIC` for this, with the following DDL:

```sql
--- IHME forecasts DDL
--- cvcb.24APR2020

CREATE OR REPLACE TABLE IHME_COVID_19 (
  DATE DATE,
  ALLBED_MEAN FLOAT,
  ALLBED_LOWER FLOAT,
  ALLBED_UPPER FLOAT,
  ICUBED_MEAN FLOAT,
  ICUBED_LOWER FLOAT,
  ICUBED_UPPER FLOAT,
  INVVEN_MEAN FLOAT,
  INVVEN_LOWER FLOAT,
  INVVEN_UPPER FLOAT,
  DEATHS_MEAN FLOAT,
  DEATHS_LOWER FLOAT,
  DEATHS_UPPER FLOAT,
  ADMIS_MEAN FLOAT,
  ADMIS_LOWER FLOAT,
  ADMIS_UPPER FLOAT,
  NEWICU_MEAN FLOAT,
  NEWICU_LOWER FLOAT,
  NEWICU_UPPER FLOAT,
  TOTDEA_MEAN FLOAT,
  TOTDEA_LOWER FLOAT,
  TOTDEA_UPPER FLOAT,
  BEDOVER_MEAN FLOAT,
  BEDOVER_LOWER FLOAT,
  BEDOVER_UPPER FLOAT,
  ICUOVER_MEAN FLOAT,
  ICUOVER_LOWER FLOAT,
  ICUOVER_UPPER FLOAT,
  LAST_UPDATED_DATE TIMESTAMP_NTZ(9) NOT NULL,
  LAST_REPORTED_FLAG BOOLEAN,
  COUNTRY_REGION VARCHAR(8192),
  ISO_3166_1 VARCHAR(2),
  ISO_3166_2 VARCHAR(2),
  PROVINCE_STATE VARCHAR(8192)
);
```

**Comment by chrisvoncsefalvay** [comment]
Closed by the merge of #216

**Comment by sfc-gh-space** [comment]
You've defined ISO_3166_2 as 2 characters but I believe it may be 3 in some countries outside of the US.
